include/vulkan/utility/vul_dispatch_table.h

```diff
@@ -0,0 +1,1770 @@
+// *** THIS FILE IS GENERATED - DO NOT EDIT ***
+// See dispatch_table_generator.py for modifications
+// Copyright 2023 The Khronos Group Inc.
+// Copyright 2023 Valve Corporation
+// Copyright 2023 LunarG, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <vulkan/vulkan.h>
+
+#include <string.h>
+
+typedef PFN_vkVoidFunction(VKAPI_PTR *PFN_GetPhysicalDeviceProcAddr)(VkInstance instance, const char *pName);
+
+// Instance function pointer dispatch table
+typedef struct VulInstanceDispatchTable_ {
```

Comment:
`VulInstanceDispatchTable_` ? why the underscore

Reply:
Thats how C types are defined - notice the `VulInstanceDispatchTable` *after* the `}`.

Reply:
I am error